solitude/settings/base.py

```diff
@@ -510,3 +510,7 @@
 
 # An arbitrary limit on the number of payment methods.
 BRAINTREE_MAX_METHODS = 5
+
+# Arbitrary amounts for top and bottom limits.
+BRAINTREE_MAX_AMOUNT = 10000
+BRAINTREE_MIN_AMOUNT = 1
```

Comment:
Do we need to specify a min/max? I suspect these defaults are not quite right, especially on the minimum